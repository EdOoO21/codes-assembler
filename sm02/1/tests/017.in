-152 652 -379 967 -252 -407 -643 568 444 442 106 354 -431 -775 879 -945 -491 -214 669 528 -141 -484 760 27 643 -350 311 733 401 935 483 -178 -719 128 -873 -714 664 621 792 -598 -691 807 445 91 144 642 956 404 -570 -323 105 -748 807 465 308 -859 -367 -161 -833 38 924 758 -30 311 228 -710
