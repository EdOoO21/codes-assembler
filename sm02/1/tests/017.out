-710
228
311
-30
758
924
38
-833
-161
-367
-859
308
465
807
-748
105
-323
-570
404
956
642
144
91
445
807
-691
-598
792
621
664
-714
-873
128
-719
-178
483
935
401
733
311
-350
643
27
760
-484
-141
528
669
-214
-491
-945
879
-775
-431
354
106
442
444
568
-643
-407
-252
967
-379
652
-152
